doc/integrator/functionality.rst

```diff
@@ -87,68 +88,12 @@ The ``vars_<project>.yaml`` file includes variables for managing *functionalitie
     then the ``functionality`` template variable will be set to a dict with one
     key/value pair: ``"<functionality_name>": ["value1","value2"]``.
 
-Usage in Templates
-~~~~~~~~~~~~~~~~~~
-
-As explained in the configuration section above,
-a functionality can be used in the Mako templates as long as it has been
-enabled using the ``functionalities:available_in_templates`` parameter in the
-``vars_<project>.yaml`` configuration file.
-
-Functionalities may be used in templates for various purposes. The examples
-below explain how to set the default basemap and how to limit access to some
-plugins according to the user's permissions.
-
-Example of the default_basemap Functionality
-............................................
-
-Using functionalities, it is easy to set the default basemap that will be
-displayed when a user loads the application depending on whether he/she is
-anonymous, authenticated or has some specific role.
-
-First make sure that ``default_basemap`` is made available in the templates
-using the ``functionalities:available_in_templates`` parameter in the
-``vars_<project>.yaml`` configuration file:
-
-.. code:: yaml
-
-    functionalities:
-        available_in_templates: [default_basemap]
-
-Then indicate (still in ``vars_<project>.yaml``) what default basemap should be used
-for anonymous users:
-
-.. code:: yaml
-
-    functionalities:
-        anonymous:
-            # some other configs...
-            default_basemap: <some_basemap>
-
-Optionally you may also indicate what basemap to use for authenticated users
-(if omitted, the anonymous ``default_basemap`` value will be used):
-
-.. code:: yaml
-
-    functionalities:
-        anonymous:
-            # ...
-        registered:
-            default_basemap: <some_other_basemap>
-
-Finally you may link ``default_basemap`` functionalities to some roles or
-users in the administration interface.
-
-So that the ``default_basemap`` is actually provided to the
-``cgxp_mapopacityslider`` plugin, use the following configuration in your
-project's ``viewer.js`` template:
-
-.. code:: javascript
+Usage in Javascript client
+~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-    {
-        ptype: "cgxp_mapopacityslider",
-        defaultBaseLayerRef: "${functionality['default_basemap'][0] | n}"
-    }
+The functionnalities will be send to the Javascript client application throw the user web crebice.
```

Comment:
what is "crebice"?

Reply:
=> service (just 3 faults in one word :-/ )